main.js

```diff
@@ -0,0 +1,34 @@
+const manager = new APIManager()
+const render = new Renderer()
+let users = []
+let i = 0;
+
+$("#load").on("click", function(){
+    manager.loadData()
+})
+
+$("#display").on("click", function(){
+    render.render(manager.data)
+})
+
+$("#save").on("click", function(){
+    const page = {...manager.data}
+    page.id = i++
+    users.push(page)
+    localStorage.users = JSON.stringify(users)
+    $(".dropdown-content").empty()
+    const source = $("#drop-template").html();
```

Comment:
you are mixing MVC try to keep the rendering in the renderer and the logic in a seperate file